docs/_includes/demos-examples-list.html

```diff
@@ -0,0 +1,14 @@
+<div class="demos columns">
+  {% for demo in page.demos %}
+    <div class="column is-6">
+      <h4 class="has-text-right">
+        {{ demo.title }}
+      </h4>
+    </div>
+    <div>
+      <a href="{{ demo.link }}" class="button demo-buttons is-medium">
```

Comment:
Until I have a chance to make a longer list of demos, please use this link: http://demo.kitware.com/histomicstk/histomicstk#?image=5bbdee4fe629140048d01adf&bounds=25823%2C19101%2C43534%2C29208%2C0

And, there should be text:

The data from the collaborative annotation study <link to the success story page> is available on a demo instance of the Digital Slide Archive.  The link goes to one of the TCGA slides that was used in the study.  If you click the ![image](https://user-images.githubusercontent.com/8781639/77785929-4e72a280-7033-11ea-9fbe-71b7fa1ce658.png) icon in the Annotations panel on the right side of the screen, you'll see the results of a collaborative annotation.

Reply:
Fixed in 8e35e4000b904e5efc5448502e24c068ad2d056e